requirements.txt

```diff
@@ -28,3 +28,4 @@ wagtail-metadata>=4.0.2,<4.1.0
 whitenoise==5.2.0
 wagtailmedia>=0.11.0,<1.0.0
 PyGithub==1.54.1
+git+https://github.com/devinit/slack_messages.git
```

Comment:
Can you investigate possibility of having our own NPM repo where we can publish this, as opposed to pulling it from Github directly?

Reply:
Also, can you investigate possibility of using (jinja2) templates to make the text more rich and formatted?